cvat-canvas3d/.gitignore

```diff
@@ -0,0 +1,2 @@
+src/*.js
```

Comment:
Why it's ignored?

Reply:
@manasars Please, answer the question

Reply:
This is same as in cvat-canvas, mostly to have only ts files.

Reply:
cvat-canvas as well as cvat-canvas3d don't have any JavaScript files. So, why do we need this line?

Reply:
In cvat-canvas this is probable outdated code.

Reply:
Resolved